### PR TITLE
Add GNU nano to text editors

### PR DESCRIPTION
**Pull request by WebSnke**

### CODE REVIEW

**Comment by KOLANICH** [comment]
nano is not secret knowledge, it is used by almost everybody who is not OK with vi and vim and emacs

**Comment by WebSnke** [comment]
> nano is not secret knowledge, it is used by almost everybody who is not OK with vi and vim and emacs

I think it's the other way around. nano is 8 years newer and a lot less popular in the [Stack Overflow Annual Developer Survey of 2015](https://insights.stackoverflow.com/survey/2015#tech-editor).

**Review by Youngfuse — APPROVED** [review]

**Review by sandra380 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -163,6 +163,7 @@ Only main chapters:
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://www.vim.org/"><b>vim</b></a> - is a highly configurable text editor.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://www.gnu.org/software/emacs/"><b>emacs</b></a> - is an extensible, customizable, free/libre text editor, and more.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://github.com/zyedidia/micro"><b>micro</b></a> - is a modern and intuitive terminal-based text editor.<br>
+&nbsp;&nbsp;:small_orange_diamond: <a href="https://github.com/zyedidia/micro](https://git.savannah.gnu.org/cgit/nano.git"><b>GNU nano</b></a> - is a small and friendly editor.<br>
```
[the remark this answers is not in the document]
Comment (WebSnke) [reply README.md:166]: > 

Why?

**Pull request closed without merging**